content/departments/engineering/dev/process/cloud_processes.md

```diff
@@ -41,3 +41,38 @@ DELETE * FROM orgs WHERE id = $ORG_ID;
 ### Other useful queries
 
 A list of useful DB queries can be found in [useful DB queries](https://docs.google.com/spreadsheets/d/1Z1-7uJwtF2etZFeqTcJS4z9WcAjKlkCxt8HQ931D3dA).
+
+## Closing down an employee's organization on Sourcegraph Cloud
+
+If you have created an organization on Sourcegraph Cloud (in production) for yourself that you’ve been using for testing or learning, follow these instructions when it’s time to delete it:
+
+- Go to your organization settings page. You can access it from the top right menu where there's a link to your org.
```

Comment:
nit: I think we should be using consistent wording at least within the same document.

```suggestion
- Go to your organization settings page. You can access it from the top right menu where there's a link to your organization.
```